docs/tr/week08/08-1.md

```diff
@@ -0,0 +1,204 @@
+
+
+<!-- Putting everything together, PIRL's NCE objective function works as follows. In a mini-batch, we will have one positive (similar) pair and many negative (dissimilar) pairs. We then compute the similarity between the transformed image's feature vector ($I^t$) and the rest of the feature vectors in the minibatch (one positive, the rest negative). We then compute the score of a softmax-like function on the positive pair. Maximizing a softmax score means minimizing the rest of the scores, which is exactly what we want for an energy-based model. The final loss function, therefore, allows us to build a model that pushes the energy down on similar pairs while pushing it up on dissimilar pairs. -->
+
+Dr. LeCun, PIRL'ın çalışması için oldukça fazla sayıda negatif örneğe ihtiyaç uyulduğunu belirtmektedir. SGD'de mini-batchlerde negatif örneklerin çok sayıda parçasını sürekli olarak tutabilmek zor olabilir. Bü  durumu çözmek için PIRL, özel bir ön bellek yapısı kullanmaktadır.
+
+<!-- Dr. LeCun mentions that to make this work, it requires a large number of negative samples. In SGD, it can be difficult to consistently maintain a large number of these negative samples from mini-batches. Therefore, PIRL also uses a cached memory bank. -->
+
+**Soru**: L2 normu yerine neden kosinüs benzerliği kullanıyoruz?
+Answer:Çünkü, oldukça kısa(merkeze yakın) ya da oldukça uzun(merkezden uzak) iki vektör seçersek bu iki vektör L2 normuna göre birbirlerine benzer diyebiliriz. Bunun sebebi, L2 normunun vektörlerin elemanları arasındaki farklarının kareleri toplamıdır. Dolayısıyla, kosinüs benzerliği kullanarak sistemimizi "hile yapmamaya" yani vektörleri uzaltıp kısaltmadan iyi bir sonuç bulmaya zorluyoruz. 
+
+<!-- **Question**: Why do we use cosine similarity instead of L2 Norm? -->
+<!-- Answer: With an L2 norm, it's very easy to make two vectors similar by making them "short" (close to center) or make two vectors dissimilar by making them very "long" (away from the center). This is because the L2 norm is just a sum of squared partial differences between the vectors. Thus, using cosine similarity forces the system to find a good solution without "cheating" by making vectors short or long. -->
+
+
+### SimCLR
+
+<center>
+<img src="{{site.baseurl}}/images/week08/08-1/fig5.png" height="75%" width="75%"/><br>
+<b>Fig. 4</b>: SimCLR'ın  ImageNet Üzerindeki Sonuçları
+</center>
+
+SimCLR diger metodlardan daha iyi sonuclar gostermektedir. Hatta, denetimli metodlarin Imagenet en iyi -1 lineer doğruluktaki performansina erismistir. 
```

Comment:
```suggestion
SimCLR önceki metodlardan daha iyi sonuçlar göstermektedir. Hatta, denetimli metodlarin Imagenet en iyi -1 lineer doğruluktaki performansina erismistir. 
```

Here comes the top-1 linear accuracy :smile: 

Maybe something like;
"Hatta, ImageNet'teki denetimli yöntemlerin seviyesine çıkmış olup 1. sıradaki doğruluğu en iyi modeldir."

Reply:
We can review it in review PR haha.